NCBI.js

```diff
@@ -0,0 +1,1111 @@
+{
+	"translatorID": "3d0231ce-fd4b-478c-b1d3-840389e5b68c",
+	"label": "NCBI",
```

Comment:
Even though we do also target books, I'd like to call this "PubMed", but let's see if someone else has something to say about this.